jax/_src/numpy/ufuncs.py

```diff
@@ -112,19 +112,116 @@ def bitwise_not(x: ArrayLike, /) -> Array:
 def invert(x: ArrayLike, /) -> Array:
   return lax.bitwise_not(*promote_args('invert', x))
 
-@implements(np.negative, module='numpy')
+
 @partial(jit, inline=True)
 def negative(x: ArrayLike, /) -> Array:
+  """Return element-wise negative values of the input.
+
+  JAX implementation of :func:`numpy.negative`.
+
+  Args:
+    x: input array or scalar.
+
+  Returns:
+    An array with same shape and dtype as ``x`` containing ``-x``.
```

Comment:
It would be worth adding a note about the behavior for unsigned integer input, as that can be somewhat confusing.

Reply:
Note on unsigned integer input is added and also added a code example on that.